index.html

```diff
@@ -67,5 +68,180 @@ <h1>
+          <h3 class="light-text">Back End Dev</h3>
+          <img src="images/counter.svg" alt="Counter icon" />
+          <h3 class="light-text">2015</h3>
+        </div>
+        <div class="card__body">
+          <p>
+            A daily selection of privately <br />
+            personalized reads; no accounts or <br />
+            sign-ups required.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </section>
+      <section class="card">
+        <img class="card__image" src="images/tonic2.svg" alt="Gym fit icon" />
+        <h2 class="card__title">Tonic</h2>
+        <div class="card__subtitle">
+          <h3>CANOPY</h3>
+          <img src="images/counter.svg" alt="Counter icon" />
+          <h3 class="light-text">Back End Dev</h3>
+          <img src="images/counter.svg" alt="Counter icon" />
+          <h3 class="light-text">2015</h3>
+        </div>
+        <div class="card__body">
+          <p>
+            A daily selection of privately <br />
+            personalized reads; no accounts or <br />
+            sign-ups required.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </section>
+      <section class="card card--two">
+        <img
+          class="card__image"
+          src="images/profprint.svg"
+          alt="Professional printing icon"
+        />
+        <h2 class="card__title">
+          Multi-Post <br />
+          Stories
+        </h2>
+        <div class="card__subtitle">
+          <h3>CANOPY</h3>
+          <img src="images/counter.svg" alt="Counter icon" />
+          <h3 class="light-text">Back End Dev</h3>
+          <img src="images/counter.svg" alt="Counter icon" />
+          <h3 class="light-text">2015</h3>
+        </div>
+        <div class="card__body">
+          <p>
+            A daily selection of privately <br />
+            personalized reads; no accounts or <br />
+            sign-ups required.
+          </p>
+          <ul class="categories">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </section>
+    </section>
+    <section class="about-section">
+      <h1>
+        About <br />
+        Myself
+      </h1>
+      <p>
+        I’m a software developer! <br />
+        I can help you build a product, feature or website. Look through some of
+        my work and experience! If you like what you see and
+        <br />
+        have a project you need coded, <br />
+        don’t hestiate to contact me.
+      </p>
+      <p>LET'S CONNECT</p>
```

Comment:
- [ ] Great job, it's one of the requirements to use `<hx>(<h2><h3>..)` for this. Kindly fix it.